core/templates/dev/head/css/oppia.css

```diff
@@ -119,6 +119,56 @@ body {
   margin-top: 15px;
 }
 
+.oppia-profile-tabs md-content {
+  background-color: white;
+}
+
+.oppia-profile-tabs md-content md-tabs {
+  background: #f6f6f6;
+  border: 1px solid #e1e1e1; 
+}
+
+.oppia-profile-tabs md-content md-tabs md-tabs-wrapper {
+  background: white; 
+}
+
+.oppia-profile-tabs md-content h1:first-child {
+  margin-top: 0; 
+}
+
+.oppia-profile-tabs md-tab {
+  background-color: #009688;
+}
+
+md-tabs-ink-bar {
```

Comment:
I've been experimenting with this, it doesn't seem to function without the !important in any of my tests. 
In this stackoverflow thread, all answers use an !important: http://stackoverflow.com/questions/33950471/how-to-change-the-underline-color-of-selected-tab-in-angular-material
It seems like it should override with a simple .oppia-profile-tabs prefix but in practice is has not done so.